gulp-tasks/jekyll.js

```diff
@@ -1,11 +1,28 @@
 'use strict';
 
+const fse = require('fs-extra');
 const gulp = require('gulp');
 const path = require('path');
 const spawn = require('child_process').spawn;
 
+
+const addNpmDependencies = () => {
```

Comment:
Good call. I originally did this because I was hoping to hook into the jekyll rebuild process, but now that I'm no longer, I agree a gulp task makes a lot more sense.